docs/MUTECT2_TUMOR_ONLY_FILTERING.md

```diff
@@ -0,0 +1,135 @@
+1000G, dbSNP, and gnomAD). Additionally, these approaches apply filters based on:
+- Clinvar's clinical severity
+- VEP's variant consequence
+- Read Depth
+- Variant Allele Frequency
+
+In addition to these approaches, the investigation also looked at the effect of providing a panel of normals and/or
+calling blacklists to Mutect2.
+
+## Benchmarking Dataset
+
+In order to evaluate these approaches, the investigation made use of a robust variant calling dataset defined here:
+https://pubmed.ncbi.nlm.nih.gov/34504347/. The data from that paper can be found here: https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/seqc/Somatic_Mutation_WG/data/.
+This dataset contains two sets of variant calling truth standards. The first of these is a synthetic truth. In this set,
+the normal sample reads have been spiked with variants. The second of these is a consensus set. This set was created
+through aligning the sample reads with multiple alignment software then calling variants in each of those alignment with
+a variety of variant calling software. Calls were then assigned confidence based on the preponderance of support from the
+matrix of calls.
+
+The truth sets for all samples can be found here:
+- Synthetic Truth
+   - Per-sample: https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/seqc/Somatic_Mutation_WG/data/DeepLearning_bams/truth_vcf/
+- Tumor-Normal Truth
+   - Per-sample, Per-software: https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/seqc/Somatic_Mutation_WG/analysis/SNVs/vcfs/WGS/
+   - All-samples, All-software (Superset): https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/seqc/Somatic_Mutation_WG/release/v1.2/
+
+## Benchmarking Approach
+
+First Mutect2 tumor-only variant calling was performed on two inputs samples:
+- Synthetic Tumor
+- Cell Line Tumor
+Both of these samples has variants called multiple times with accompanying inputs:
+- With/Without Panel of Normals (both Genomic Data Commons (GDC) and Mutect2 1000G)
+- With/Without Calling Blacklist
+
+The resultant variant files were then passed through the filters described above and compared against their respective
+truth sets. The synthetic tumor variants were compared against the per-sample synthetic truth VCF. The cell line tumor-only
+variants were compared against the sample's BWA-Mutect2 tumor-normal results. The comparison was done on a by-site basis.
+Variants were considered a match if they matched all: chromosome, position, reference allele, and alternative allele.
+
+## Recommendations from Benchmarking
+
+Tumor-only calling with Mutect2 is inherently very noisy. Therefore, most of our recommendations revolve around reducing
+the noise where possible. These recommendations are:
+- Restrict the callable regions with a blacklist and GDC Panel of Normals (PON)
+- Filter low support reads:
+   - Allele Depth (AD) > 0: WGS uninformative reads
+   - Variant Allele Frequency (VAF) > 1%: WXS noise
+- Filter potential germline variants with gnomAD AF < 0.00003
```

Comment:
So, to be clear, not `0.003`, but `0.0003`? We were using I think `0.001` before which was from somatic criteria

Reply:
NM, I reads further, and you really meant this

Reply:
I'll try and clear this up. I guess what I'm trying to say is "get rid of variants that have gnomAD AF > 0.00003"

Reply:
Done